specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Jobs.json

```diff
@@ -1075,12 +1075,75 @@
             "template": {
               "description": "Job's execution container.",
               "$ref": "#/definitions/JobExecutionTemplate"
+            },
+            "detailedStatus": {
+              "description": "Detailed status of the job execution.",
+              "$ref": "#/definitions/ExecutionJobStatus"
             }
           },
           "x-ms-client-flatten": true
         }
       }
     },
+    "ExecutionJobStatus": {
+      "description": "Container Apps Job execution.",
+      "type": "object",
+      "properties": {
+        "pods": {
+          "description": "Pods in the execution.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExecutionPodStatus"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "ExecutionPodStatus": {
+      "description": "Container Apps Job execution.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Pod Name.",
+          "type": "string"
+        },
+        "containers": {
+          "description": "Containers in the execution",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExecutionContainerStatus"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "ExecutionContainerStatus": {
+      "description": "Container Apps Job execution.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Container Name.",
+          "type": "string"
+        },
+        "code": {
+          "description": "Exit code",
+          "type": "integer",
+          "format": "int32"
+        },
+        "reason": {
+          "description": "Container reason",
```

Comment:
```suggestion
          "description": "Error reason",
```

Reply:
It may not be an error

Reply:
What is it meant to say? Could you add the same to description

Reply:
@lihaMSFT based on the example added:
![image](https://github.com/Azure/azure-rest-api-specs/assets/29655676/094ece00-7405-4eb5-8243-5fc1c95737ae)

Instead of `ContainerReason` can this be `ContainerState`?